packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -392,6 +397,33 @@ suite('lit-html', () => {
     test('text after comment', () => {
       assertRender(html`<!-- -->${'A'}`, '<!-- -->A');
     });
+
+    test('renders after existing content', () => {
+      container.appendChild(document.createElement('div'));
+      assertRender(html`<span></span>`, '<div></div><span></span>');
+    });
+
+    test('renders before `renderBefore`, if specified', () => {
+      container.appendChild(document.createElement('div'));
+      assertRender(html`<span></span>`, '<span></span><div></div>', {
+        renderBefore: container.firstChild,
+      });
+    });
+
+    test('renders same template before different `renderBefore` nodes', () => {
```

Comment:
I think this test needs an update to catch the problem with storing the new part. The current code will create a new part every render.

Reply:
Done.